tests/python/unittest/test_pass_loop_partition.py

```diff
@@ -365,6 +365,29 @@ def test_conv_tiling():
     stmt = tvm.ir_pass.Simplify(stmt)
     assert(not any(collect_visit(stmt, lambda x: isinstance(x, tvm.stmt.IfThenElse))))
 
+def test_multilevel_splitting_with_indivisble_factors():
+    import topi
+    A = tvm.placeholder((130, 54), dtype="float32")
+    B = topi.nn.relu(A)
+    s = tvm.create_schedule(B.op)
+    (y, x) = s[B].op.axis
+    (xo, xi) = s[B].split(x, factor=2)
+    (yo, yi) = s[B].split(y, factor=8)
+    (xoo, xoi) = s[B].split(xo, factor=16)
+    (yoo, yoi) = s[B].split(yo, factor=16)
+    s[B].reorder(yoo, xoo, yoi, xoi, yi, xi)
```

Comment:
Can we have a simpler example to show case the fix? I still don't understand the problem description.

Reply:
Updated example.